src/main/java/io/github/moulberry/notenoughupdates/recipes/MobLootRecipe.java

```diff
@@ -64,7 +65,14 @@ public ItemStack getItemStack() {
 			if (itemStack == null) {
 				itemStack = drop.getItemStack().copy();
 				List<String> arrayList = new ArrayList<>(extra);
-				arrayList.add("§r§e§lDrop Chance: §6" + chance);
+				try {
+					String chanceText = chance.substring(0, chance.length() - 1);
+					int chanceIn = (int) (100.0 / Double.parseDouble(chanceText));
+					String format = GuiProfileViewer.numberFormat.format(chanceIn);
+					arrayList.add("§r§e§lDrop Chance: §61/" + format + " (" + chance + ")");
+				} catch (Exception e) {
```

Comment:
if you blanket catch an exception, please call e.printEtackTrace(). if you expect a specific failure mode, then narrow your exception scope, or other legitimate failures might not show up

Reply:
Good call! Thanks. Will change it now